backend-training/PREWORK.md

```diff
@@ -0,0 +1,63 @@
+# Icalia Labs Dojo - Backend Prework
+
+In order to be successful while training for Backend using the Dojo guides we provide, we put together a set of resources, tutorials and tools to get you started, and this will also help standarize the knowledge among members.
+
+### Concepts to be aware of
+
+<ul>
+  <li><b>Basic object oriented programming</b></li>
+  <ul>
+    <li>What is an <a target="_blank" href="http://en.wikipedia.org/wiki/Object_(computer_science)">Object?</a></li>
+    <li>What is a <a target="_blank" href="http://en.wikipedia.org/wiki/Class_(computer_science)">Class?</a></li>
+    <li>What is <a target="_blank" href="http://en.wikipedia.org/wiki/Encapsulation_(object-oriented_programming)">encapsulation</a>, 
+    <a target="_blank" href="http://en.wikipedia.org/wiki/Object_composition#Aggregation">aggregation</a>,
+    <a target="_blank" href="http://en.wikipedia.org/wiki/Object_composition#Composite_types_in_C">composition?</a></li>
+    <li>What is <a target="_blank" href="http://en.wikipedia.org/wiki/Inheritance_%28object-oriented_programming%29">inheritance</a>?</li>
+    <li>What is an <a target="_blank" href="http://en.wikipedia.org/wiki/Class_%28computer_programming%29">abstract class?</a></li>
+    <li>Difference between a <a target="_blank" href="http://en.wikipedia.org/wiki/Class_%28computer_programming%29#The_concept_of_class_interface">Interface</a> and an <a href="http://en.wikipedia.org/wiki/Class_%28computer_programming%29">abstract class?</a></li>
+    <li>What is a <a target="_blank" href="http://en.wikipedia.org/wiki/Class_diagram">Class Diagram?</a></li>
+    <li>Check this <a target="_blank" href="http://en.wikipedia.org/wiki/Data_structure">data structures</a></li>
+  </ul>
+</ul>
+
+### Software requirements
+
+We provide a list of necessary software you need to install before starting your training.
```

Comment:
We need to add a note that says "if your current OS is windows you can setup a linux vm  or install linux in another partition,  windows is not allowed in our development workflow  at the moment

Reply:
Good one! Let me add it!

Reply:
Done!